settings.gradle.kts

```diff
@@ -1,12 +1,15 @@
-rootProject.name = "reflekt"
-
-include(":reflekt-plugin")
-
 pluginManagement {
     repositories {
         mavenCentral()
         gradlePluginPortal()
     }
 }
 
+rootProject.name = "reflekt"
+
+includeBuild("build-logic")
+include(":reflekt-plugin")
 includeBuild("using-embedded-kotlin")
+
+enableFeaturePreview("STABLE_CONFIGURATION_CACHE")
```

Comment:
Please explain how this preview helps.

Reply:
I added this because it improves the configuration cache experience, at least for me; doesn't need to be in this PR, though. I'll remove it.